src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/LanguageServer/RazorSemanticTokensBenchmark.cs

```diff
@@ -91,53 +91,55 @@ public async Task RazorSemanticTokensRangeAsync()
                 textDocumentIdentifier, Range, DocumentContext, cancellationToken).ConfigureAwait(false);
         }
 
+        private static LspServices GetLspServices()
```

Comment:
Do we need this? From what I can tell, this method is unused (unless I'm missing something).